pkgs/applications/science/math/almonds/default.nix

```diff
@@ -0,0 +1,28 @@
+{ pythonPackages, fetchFromGitHub, ncurses }:
+
+pythonPackages.buildPythonPackage {
+  name = "almonds-2015-12-27";
```

Comment:
What's wrong with the last stable release?
Also please split this up into `pname` and `version` and leave out `name`.

Reply:
Actually, I think the latter would only make sense when using `buildPythonPackage`. Sorry :)